source/settings.cpp

```diff
@@ -1768,6 +1801,18 @@ namespace Settings {
       Keysanity.SetSelectedIndex(3); //Set small keys to any dungeon so FiT basement door will be locked
       GossipStoneHints.SetSelectedIndex(0);
     }
+
+    InitMusicRandomizer();
+    if (ShuffleMusic) {
+      if (ShuffleBGM)
+        ShuffleSequences(SeqType::SEQ_BGM);
+      if (ShuffleFanfares.Is(1)) {
+        ShuffleSequences(SeqType::SEQ_FANFARE);
+        if (ShuffleOcaMusic)
+          ShuffleSequences(SeqType::SEQ_OCARINA);
+      } else if (ShuffleFanfares.Is(2))
+        ShuffleSequences(SeqType::SEQ_FANFARE | SeqType::SEQ_OCARINA);
+    }
```

Comment:
This needs to be redone because Ocarina Music won't be shuffled if Fanfares are off:
![image](https://user-images.githubusercontent.com/5352197/128326606-7bd3a051-4042-4c72-b526-bcb5a15ad08d.png)